src/pages/Thanks/ThanksPage.vue

```diff
@@ -208,6 +210,26 @@ export default {
 	},
 	computed: {
 		selectedLoan() {
+			if (this.categoryShareVersion === 'a' || this.categoryShareVersion === 'b') {
+				const loans = [...this.loans];
+				loans.sort((a, b) => {
```

Comment:
Doesn't need to change for this PR, but this sort code is a great example of a chunk of code with very specific expectations that could be extracted to a util and unit tested.